tests/unit/keyinput/test_basekeyparser.py

```diff
@@ -342,3 +344,13 @@ def test_respect_config_when_matching_counts(keyparser, config_stub):
 
     assert not keyparser._sequence
     assert not keyparser._count
+
+
+def test_count_limit_exceeded(handle_text, keyparser, caplog):
+    max_digits = sys.get_int_max_str_digits()
+    keys = (max_digits + 1) * (Qt.Key.Key_1,)
+
+    with caplog.at_level(logging.ERROR):
+        handle_text(keyparser, *keys, Qt.Key.Key_B, Qt.Key.Key_A)
+
+    assert f"Count limit exceeded ({max_digits} digits)" in caplog.text
```

Comment:
```suggestion
    assert f"Count limit exceeded ({max_digits} digits)" in caplog.messages
```

so it actually tests it as a string in a list, instead of a substring match in a stream of text. Seems a bit cleaner and stricter.